src/repositories/RemittanceOutwardsRepository.js

```diff
@@ -96,6 +96,13 @@ export const RemittanceOutwardsRepository = {
     reopen: service + 'reopenOWM',
     close: service + 'closeOWM'
   },
+  OutwardsReturn: {
+    qry: service + 'qryOWR',
+    page: service + 'pageOWR',
```

Comment:
page endpoint never used, ask fawzi if he can add page endpoint for OWR else remove it